packages/gluestack-ui/src/r5/feedback/fhir-query-loader.tsx

```diff
@@ -0,0 +1,62 @@
+import {
+  FhirError,
+  FhirErrorProps,
+  FhirQueryLoaderRendererProps,
+} from "@bonfhir/react/r5";
+import { Box, Center, Spinner } from "@gluestack-ui/themed";
+import { ComponentProps, ReactElement } from "react";
+
+type BoxProps = ComponentProps<typeof Box>;
+type CenterProps = ComponentProps<typeof Center>;
+type SpinnerProps = ComponentProps<typeof Spinner>;
+
+export function GlueStackFhirQueryLoader(
+  props: FhirQueryLoaderRendererProps<GlueStackFhirQueryLoaderProps>,
+): ReactElement | null {
+  if (props.isLoading) {
+    return (
+      <Center style={props.style} {...props.rendererProps?.center}>
```

Comment:
I wonder if the top-level component should be the same for all conditions (namely a Box)?
I am not sure about the consequences for layout...
We can always adjust later.

Reply:
yeah,I mainly followed the Mantine component, which has a center for the loading, and a <Stack /> for the others. Glustack currently has a VStack and HStack, but I didn't see a Stack (which Native Base had), so I just went with Box for now.